projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-popover/index.tsx

```diff
@@ -90,7 +90,7 @@ export const AiAssistantPopover = ( {
 
 	const isLoading = requestingState === 'requesting' || requestingState === 'suggesting';
 
-	const placeholder = __( 'Which form do you need?', 'jetpack' );
+	const placeholder = __( 'Ask Jetpack AI to create your form.', 'jetpack' );
```

Comment:
I haven't seen many placeholders in the Jetpack plugin with the period `.` at the end of the string.  Something that takes my attention because we add other ones like `?`, `!`, `…`, etc...

I found just a few, like [this one](https://github.com/Automattic/jetpack/blob/2f1a5adcfb38e216f9d43e6ffe838f66b178552e/projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-popover/index.tsx#L95), but in general, we don't add it.